doc/CORTX_on_Open_Virtual_Appliance.rst

```diff
@@ -172,15 +172,15 @@ The procedure to install CORTX on OVA is mentioned below.
 
     systemctl start|restart <service_name>
 
-#. Run **ip a l** and record the IP addresses of the following interfaces:
+#. Run **ip a l** and record the IP addresses of **ens32 + ens33** :
 
-   * ens32 - Management IP
```

Comment:
I believe this info is still important because, in step 13, it says "Using the management IP..." 
If we remove `ens32 - Management IP`, we don't know which one is the *management IP*.